src/snowflake/snowpark/dataframe.py

```diff
@@ -1464,6 +1808,27 @@ def alias(self, name: str):
         Args:
             name: The alias as :class:`str`.
         """
+
+        # TODO: Last doctest seems broken, this is an experimental feature. Should we remove it?
+
```

Comment:
The merge gates seems to be working fine without the test being skipped.

Reply:
removed comment.

Reply:
cf. https://github.com/snowflakedb/snowpark-python/pull/2506